drivers/usb/device/usb_dc_stm32.c

```diff
@@ -545,21 +546,25 @@ int usb_dc_ep_configure(const struct usb_dc_ep_cfg_data * const ep_cfg)
 	u8_t ep = ep_cfg->ep_addr;
 	struct usb_dc_stm32_ep_state *ep_state = usb_dc_stm32_get_ep_state(ep);
 
-	LOG_DBG("ep 0x%02x, ep_mps %u, ep_type %u", ep_cfg->ep_addr,
-		ep_cfg->ep_mps, ep_cfg->ep_type);
+	LOG_DBG("ep 0x%02x, previous ep_mps %u, ep_mps %u, ep_type %u",
+		ep_cfg->ep_addr, ep_state->ep_mps, ep_cfg->ep_mps,
+		ep_cfg->ep_type);
 
 	if (!ep_state) {
 		return -EINVAL;
 	}
 
 #ifdef USB
-	if (DT_USB_RAM_SIZE <=
-	    (usb_dc_stm32_state.pma_offset + ep_cfg->ep_mps)) {
-		return -EINVAL;
+	if (ep_cfg->ep_mps > ep_state->ep_pma_buf_len) {
```

Comment:
Why do we need `ep_state->ep_pma_buf_len`? Isn't enough to check `if (ep_cfg->ep_mps > ep_state->ep_mps)`?

Reply:
Ignore my comment. We need it because `ep_cfg->ep_mps` may be less than `ep_state->ep_mps`.